### PR TITLE
Infrastructure needed for Manifest Services converted to Rails Application

### PR DESCRIPTION
**Pull request by orangewolf**

### CODE REVIEW

File: templates/iiif-manifest-compose.ecs.yml [hunk 1]
```diff
@@ -2,8 +2,11 @@ version: '3'
 services:
   iiif_manifest:
     environment:
-      IIIF_MANIFESTS_BASE_URL: 'https://$$host/manifests/'
+      HONEYBADGER_API_KEY_MANAGEMENT: ${HONEYBADGER_API_KEY_MANIFEST}
```
Comment (fnibbit) [inline templates/iiif-manifest-compose.ecs.yml:5]: Do we want a separate honeybadger api key for the manifest service?  We probably don't want to call it "management" inside the container either way.

File: templates/iiif-manifest-compose.ecs.yml [hunk 2]
```diff
@@ -2,8 +2,11 @@ version: '3'
...
       IIIF_IMAGE_BASE_URL: 'https://$$host/iiif'
+      IIIF_MANIFESTS_BASE_URL: 'https://$$host/manifests/'
```
Comment (fnibbit) [inline templates/iiif-manifest-compose.ecs.yml:7]: I don't think this trick works here.

File: templates/iiif-manifest-compose.yml [hunk 3]
```diff
@@ -3,8 +3,16 @@ services:
...
     ports:
-      - '80:80'
+      - '3002:3002'
     environment:
-      IIIF_MANIFESTS_BASE_URL: http://localhost/manifests/
+      AWS_ACCESS_KEY_ID:
+      AWS_DEFAULT_REGION: us-east-1
+      AWS_SECRET_ACCESS_KEY:
       IIIF_IMAGE_BASE_URL: http://localhost:8182/iiif
+      IIIF_MANIFESTS_BASE_URL: http://localhost/manifests/
```
Comment (fnibbit) [inline templates/iiif-manifest-compose.yml:12]: :3002?

File: templates/iiif-manifest-compose.ecs.yml (hunk 1 above)
Comment (orangewolf) [reply templates/iiif-manifest-compose.ecs.yml:5]: yep, that's just a typo. good catch

**Comment by maxkadel** [comment]
superceded

**Pull request closed without merging**